superset-frontend/src/common/components/index.tsx

```diff
@@ -144,12 +155,9 @@ export const StyledNav = styled(AntdMenu)`
       }
     }
   }
-`;
-
-export const StyledSubMenu = styled(AntdMenu.SubMenu)`
-  color: ${({ theme }) => theme.colors.grayscale.dark1};
-  border-bottom: none;
   .ant-menu-submenu-title {
+    position: relative;
+    top: -7px;
```

Comment:
There are three 7px values in this PR's diff... do they work as 8px so we can use gridUnit, or is there something like a 1px line that we're dancing around?

And if that is the case, can we add one more `gridUnit` and subtract 1px? Then if we ever make a gridUnit 5px or something, these three styles won't break.